packages/htm/README.md

```diff
@@ -18,34 +18,36 @@ npm install @remote-ui/htm --save
 
 ## Usage
 
-`@remote-ui/htm` provides two functions. The first, `htm`, is a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which binds the main function from the `htm` library into a basic object structure. `render`, the second function exported from this library, takes this generated structure and a [`@remote-ui/core` `RemoteRoot`](../core#remoteroot), and appends all of the resulting components into that root.
+`@remote-ui/htm` provides two functions. The first, `createRender`, is a factory function that takes a [`@remote-ui/core` `RemoteRoot`](../core#remoteroot), and returns a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates). This tagged template binds the main function from the `htm` library into creating a tree of `@remote-ui/core` `RemoteComponent` and `RemoteText` objects. The second function, `append`, is a helper to call `appendChild()` on a `RemoteRoot` or `RemoteComponent` for each node in the tree returned by the tagged template.
 
 ```ts
 import {createRemoteRoot} from '@remote-ui/core';
-import {htm, render} from '@remote-ui/htm';
+import {createRender, append} from '@remote-ui/htm';
```

Comment:
I agree that the naming is kind of wonky/ a little all over the place. Ultimately, I wanted to stay away from the explicit `htm` naming in this round because, as I was reading it, it made this library sound too related to HTML, when in fact it has no relationship to that at all. `createRender` was the name that felt best to me (`createUI` made it sound like it was actually creating UI, which it isn't — it's just a factory to create a function that can help with creating UI), and `ui` was the tagged template that felt the most right to me (`render` is very confusing since `@remote-ui/react`/`mini-react` both have a `render` function for starting React's rendering loop). I will give it a bit more thought, though.